freeipa_community_portal/config.py

```diff
@@ -18,24 +18,52 @@
 # along with this program.  If not, see <http://www.gnu.org/licenses/>.
 
 import ConfigParser
+import errno
 import os
 
+from . import PACKAGE_DATA_DIR
+
 
 class Config(object):
     default_configs = [
         '/etc/freeipa_community_portal.ini',
-        'conf/freeipa_community_portal_dev.ini'
+        os.path.join(PACKAGE_DATA_DIR,
+                     'conf/freeipa_community_portal_dev.ini'),
```

Comment:
Bummer :-1: 
I've to find a better and more clean way to load the development configuration under development and test conditions.